newsrevenuehub/rev-engine
# [NRH-361] OrgAdmin page templates

#### What's this PR do?
Implements template creation and template use from the Org Admin.

#### How should this be manually tested?
From the page edit view, notice a new button that you can use to create a template from the existing page. Try it out.
Then, from the page-create modal, notice a new optional dropdown for selecting a template from which to make your new page. Try that out too!

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No
